### ISSUE TITLE
BEC API

### ISSUE THREAD

**Issue by joeyklee** [joeyklee/bec-explorer#11]
Before continuing with dev. I think I will try to structure our function calls as a modular SQL API as shown here: https://docs.cartodb.com/cartodb-platform/sql-api/making-calls/

**Comment by cmahony** [comment]
yes, i've been wondering about how to do the functions and build the graphs. I have no familiarity with SQL and i have been planning on writing blocks of R code for each graph. if i did really good annotation, would this be a viable workflow? i.e., you would translate into SQL. if not, how much of a learning curve is it for me to learn SQL?

**Comment by joeyklee** [comment]
@cmahony - so the sql stuff is more a way to get back the data from cartodb and make sure the map and the charts are all updating together, etc. I would hold off on writing any R code unless you are going to be doing that for your research anyways. I'm going to try to make things a bit easier for us to work with our charts/data, etc.

**Comment by cmahony** [comment]
@joeyklee ok what i'll do is draft up a few different chart types in R to put some ideas forward.

**Comment by joeyklee** [comment]
Turns out anyone can access the data using the cartodb sql interface

I think I will write some API endpoints to make it easier for people to get the data and post them on the API page.
